rpc/client.go

```diff
@@ -0,0 +1,90 @@
+package rpc // import "collectd.org/rpc"
+
+import (
+	"io"
+	"log"
+
+	"collectd.org/api"
+	pb "collectd.org/rpc/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+// client is a wrapper around pb.CollectdClient implementing Interface.
```

Comment:
Usually doc comments should be full sentences, hence this would be "A client …".